lua/tabnine/user_commands.lua

```diff
@@ -1,18 +1,34 @@
 local M = {}
 local api = vim.api
-local fn = vim.fn
 local tabnine_binary = require("tabnine.binary")
 local status = require("tabnine.status")
+local config = require("tabnine.config")
 
 function M.setup()
-	api.nvim_create_user_command("TabnineHub", function()
-		tabnine_binary:request({ Configuration = { quiet = false } }, function() end)
+	if not config.is_enterprise() then
+		api.nvim_create_user_command("TabnineHub", function()
+			tabnine_binary:request({ Configuration = { quiet = false } }, function() end)
+		end, {})
+
+		api.nvim_create_user_command("TabnineHubUrl", function()
+			tabnine_binary:request({ Configuration = { quiet = true } }, function(response)
+				print(response.message)
+			end)
+		end, {})
+	else
+		api.nvim_create_user_command("TabnineWhoAmI", function()
+			tabnine_binary:request({ UserInfo = { quiet = false } }, function(response)
+				print(response.email)
```

Comment:
suggest to add info if part of a team and if admin/member, for easier debugging of problems.